bitfield/query.py

```diff
@@ -24,23 +24,19 @@ def get_prep_lookup(self):
 
 
 class BitQuerySaveWrapper(BitQueryLookupWrapper):
-    def as_sql(self, qn, connection):
+    def as_sql(self, compiler, connection):
```

Comment:
This change seems like it should result in changes to the tests; perhaps it's worth adding a test for this?